app/src/main/java/com/onboarding/calculatorkotlin/util/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.onboarding.calculatorkotlin.util
+
+const val EMPTY_STRING = ""
+const val MINUS = "-"
+const val ONE_VALUE_REMOVED = 1
```

Comment:
this contants it used only in the Model, Please put on the componion object of this class

Reply:
+1

Reply:
Solved!